Installation from AWS Lambda Console (SAM Template) code outdated

<!--
Before submitting an issue, please search for existing issues and
make sure it is not a duplicate.
-->

# Promote-or-quarantime-plugin SAM Deployment
https://github.com/trendmicro/cloudone-filestorage-plugins/tree/master/post-scan-actions/aws-python-promote-or-quarantine#from-aws-lambda-console

## Describe the bug
the template used to deploy directly on the SAM has a Lambda with a syntax error in line **147** and **148**
apparently that template is taking an old version of the script

the current code is this
`
f '{FSS_TAG_PREFIX} scan-detail-code': str (code),
f '{FSS_TAG_PREFIX} scan-detail-message': urllib.parse.quote_plus (CODE_MESSAGES.get (code, CODE_MESSAGES [CODE_MISC])),
`
and it should be this
`
f '{FSS_TAG_PREFIX} scan-detail-cod= {str (code) }',
f '{FSS_TAG_PREFIX} scan-detail-message = { urllib.parse.quote_plus (CODE_MESSAGES.get (code, CODE_MESSAGES [CODE_MISC])) }',
`
error code from CloudWatch
`
[ERROR] Runtime.UserCodeSyntaxError: Syntax error in module 'handler': invalid syntax (handler.py, line 148)
Traceback (most recent call last):
  File "/var/task/handler.py" Line 148
                f'{FSS_TAG_PREFIX}scan-detail-message': urllib.parse.quote_plus(CODE_MESSAGES.get(code, CODE_MESSAGES[CODE_MISC])),
`
## To reproduce
Steps to reproduce the behavior:
1. Go to https://console.aws.amazon.com/lambda/home?#/create/app?applicationId=arn:aws:serverlessrepo:us-east-1:415485722356:applications/cloudone-filestorage-plugin-action-promote-or-quarantine 
2. Complete the parameters with S3 to configure and Deploy App
3. Upload File in S3, verify this is Scanned by FSS
4. The Lambda PromoteOrQuarantine is Invoked but the file isn't move 
5. Review the Cloud Wath Log Stream and see the Error

## Additional context

Check the repository handler.py and you do not have this error, it seems that the SAM Template that is being used does not have the updated code

this is the codeUri that is in the template
CodeUri:
         Bucket: awsserverlessrepo-changesets-plntc6bfnfj
         Key: 846753579733 / arn: aws: serverlessrepo: us-east-1: 415485722356: applications-cloudone-filestorage-plugin-action-promote-or-quarantine-versions-1.0.5 / b3d5c889-6ebc-42bb-8a76-6c89f9df43f8

--- a/post-scan-actions/aws-python-promote-or-quarantine/template.yml
+++ b/post-scan-actions/aws-python-promote-or-quarantine/template.yml
@@ -13,7 +13,7 @@ Metadata:
     ReadmeUrl: README.md
     Labels: [trendmicro, cloudone, filestorage, s3, bucket, plugin, promote, quarantine]
     HomePageUrl: https://github.com/trendmicro/cloudone-filestorage-plugins
-    SemanticVersion: 1.0.5
+    SemanticVersion: 1.0.6
     SourceCodeUrl: https://github.com/trendmicro/cloudone-filestorage-plugins/tree/master/post-scan-actions/aws-python-promote-or-quarantine
 
 Parameters:

